.env-example

```diff
@@ -8,7 +8,6 @@ GOOGLE_API_CX_CY=xxxx:xxxx
 STATSD_HOST=localhost
 STATSD_PORT=8125
 WEBCHAT_DOMAIN=xxxx.xxxx
-FRONTEND_HTTP_REQUEST_TIMEOUT=xx
```

Comment:
Good point. This had to be removed as this `FRONTEND_HTTP_REQUEST_TIMEOUT=10` should be left as is without making changed.